scripts/rollup/build.js

```diff
@@ -503,44 +507,68 @@ function createBundle(bundle, bundleType) {
     }
   }
 
-  const filename = getFilename(bundle.name, bundle.hasteName, bundleType);
+  const filename = getFilename(
+    bundle.output || bundle.entry,
```

Comment:
Looks like this only exists to map `react-dom/server` to `react-dom/server.node`

Couldn't we simplify by just renaming `react-dom/server.js` to `react-dom/server.node.js`?

Reply:
Probably yes. I wanted to avoid an explicit separate entry point because it feels like public API. But I guess we have `.browser` anyway so might as well do that.